pkg/notifier/github/commits.go

```diff
@@ -9,13 +9,21 @@ import (
 // methods of GitHub API
 type CommitsService service
 
-func (g *CommitsService) MergedPRNumber(ctx context.Context, sha string) (int, error) {
+type PullRequestState string
+
+const (
+	PullRequestStateOpen   = "open"
+	PullRequestStateClosed = "closed"
+	PullRequestStateAll    = "all"
+)
+
+func (g *CommitsService) PRNumber(ctx context.Context, sha string, state PullRequestState) (int, error) {
 	prs, _, err := g.client.API.PullRequestsListPullRequestsWithCommit(ctx, sha, nil)
 	if err != nil {
 		return 0, err
 	}
 	for _, pr := range prs {
-		if pr.GetState() != "closed" {
+		if pr.GetState() != string(state) {
```

Comment:
Could you check Pull Request's `UpdatedAt`?

> How to handle when multiple pull requests are associated
> Choose the latest updated pr

https://github.com/suzuki-shunsuke/tfcmt/issues/387#issuecomment-1206353018

I don't know the order of `prs`

Reply:
Done: [`d359a60` (#390)](https://github.com/suzuki-shunsuke/tfcmt/pull/390/commits/d359a60fbd9a02da655b75864565efa3b149e9ca)